### DISCUSSION TITLE
Join the team, please

### DISCUSSION THREAD

**Discussion by matheucomth** [ROM-builders/temporary#10419] [General]
https://cirrus-ci.com/build/6309574247448576
https://cirrus-ci.com/build/4518611673939968

dt
https://github.com/matheucomth/device_xiaomi_tulip

**Comment by Apon77** [comment]
> https://cirrus-ci.com/build/6309574247448576 https://cirrus-ci.com/build/4518611673939968
> 
> dt https://github.com/matheucomth/device_xiaomi_tulip

give telegram id

**Reply by matheucomth** [reply]
@matheucomth
